radish/documentors/__main__.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module provides a script to document Radish step implementations
+"""
+
+from sys import exit
+
+from os.path import split
+
+from docopt import docopt
+
+from radish import __VERSION__
+
+from radish.documentors.sphinx.multipage import MultiPageSphinxDocumentSet
+from radish.documentors.sphinx.singlepage import SinglePageSphinxDocument
+from radish.documentors.steps_writer import StepsDocumentWriter
+
+from radish.errororacle import error_oracle, enable_excepthook
+
+from radish.exceptions import RadishError
+
+from radish.loader import load_module_assets
+
+from radish.main import setup_config
+
+from radish.stepregistry import StepRegistry
+
+
+# ........................................................................... #
+@error_oracle
+def main():
+    usage = """
+Usage:
+    radish-docs sphinx <module-path>
```

Comment:
> (though it is not strictly module since it does not check for init.py in folders)

You are mixing up packages (a folder containing a \_\_init\_\_.py) and modules (which are just regular *.py files).

Thus, the name is actually fine. (Or rename it to `<package-path>`

> as in plural, since step libraries will be shared and necessarily be in the basedir.

This is kind of what #40 is about. I'm not quiet sure what's best there. Because we would end up having multiple *basedirs* which could require a renaming any way. 
Maybe `<radish-paths...>` ? That would also include the *terrain* files? Which we might want to have documented in the future as well?